Compiler.coffee

```diff
@@ -70,8 +70,12 @@ class Compiler
         props.className = className
 
       React.cloneElement child, props,
-        React.Children.map child.props.children, (child) =>
-          @traverseChild child, parentClass
+        if React.Children.count(child.props.children) is 1
```

Comment:
Usually, React children is array, but also could be object, null or undefined in case there is the only one child (just to save a little bit of the memory), so It should work fine.